modules/installing-private-image-registry-private-azure.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+//* registry/configuring_registry_storage-azure.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-private-image-registry-private-azure"]
+= Optional: Preparing a private Microsoft Azure cluster for a private image registry
+
+By installing a private image registry on a private Microsoft Azure cluster, you can create private endpoints. Private endpoints disable public facing endpoints to the internal registry, adding an extra layer of security to your {product-title} deployment.
+
+Use the following guide to prepare your private Microsoft Azure cluster for installation with a private image registry.
+
+.Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
+
+* You have installed the OpenShift CLI (oc).
+
+* You have prepared an `install-config.yaml` that includes the following information:
+** A file that includes your VPC configuration
```

Comment:
what file is this? I thought networking (vpc) configuration was part of the install config? 🤔

Reply:
I really don't remember where this came from. I've removed the following line:

* A file that includes your VPC configuration

If the line is required I will re-add it.